curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/65824872894f59546e3084e2.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-25
 
 The `while` loop you created in the previous step is an infinite loop that will allow the program to continuously present menu options until the user decides to exit. 
 
-After the `print()` call, add another one to print the string `1. Add an expense`.
+After the `print()` call, add another one to print the string `'1. Add an expense'`.
```

Comment:
When you add this, you want them to print the single quotation marks, which I believe is supposed to print `1. Add an expense` since it's supposed to print the string '1. Add an expense' but not `'1. Add an expense'`

Reply:
No, the quotes should not be printed, they are there to indicate that is a string